Rakefile

```diff
@@ -1,52 +1,123 @@
-########################################################################################################################
-#  BuildingSync®, Copyright (c) 2015-2019, Alliance for Sustainable Energy, LLC.
-#  All rights reserved.
-#
-#  Redistribution and use in source and binary forms, with or without modification,
-#  are permitted provided that the following conditions are met:
-#
-#  (1) Redistributions of source code must retain the above copyright notice,
-#  this list of conditions and the following disclaimer.
-#
-#  (2) Redistributions in binary form must reproduce the above copyright notice,
-#  this list of conditions and the following disclaimer in the documentation and/or
-#  other materials provided with the distribution.
-#
-#  (3) Neither the name of the copyright holder nor the names of any contributors
-#  may be used to endorse or promote products derived from this software without
-#  specific prior written permission from the respective party.
-#
-#  (4) Other than as required in clauses (1) and (2), distributions in any form of
-#  modifications or other derivative works may not use the "BuildingSync" trademark or
-#  any other confusingly similar designation without specific prior written permission
-#  from Alliance for Sustainable Energy, LLC.
-#
-#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDER(S) AND ANY CONTRIBUTORS "AS IS" AND
-#  ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
-#  WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
-#  IN NO EVENT SHALL THE COPYRIGHT HOLDER(S), ANY CONTRIBUTORS, THE UNITED STATES
-#  GOVERNMENT, OR THE UNITED STATES DEPARTMENT OF ENERGY, NOR ANY OF THEIR EMPLOYEES,
-#  BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
-#  DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
-#  LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
-#  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
-#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN
-#  IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-########################################################################################################################
+require 'bundler'
```

Comment:
We updated the Copyrights in the develop branch. Make sure to merge develop up sooner than later.

Reply:
Develop branch merged into DA branch. [related commit](https://github.com/BuildingSync/BuildingSync-gem/commit/f68a2dd3f2e448eec74ad55f4c7f440adc1f4855)
Updated/added copyright in all files. [related commit](https://github.com/BuildingSync/BuildingSync-gem/commit/d5e8144a3f6e04493f370dbd7b25eb853e0f2a2c)